pcstac/Dockerfile.dev

```diff
@@ -1,5 +1,8 @@
 FROM pc-apis-stac
 
+COPY setup.cfg /opt/src/setup.cfg
 COPY requirements-dev.txt /opt/src/requirements-dev.txt
 
 RUN pip install -r /opt/src/requirements-dev.txt
+
+RUN pip install pypgstac==0.4.2
```

Comment:
I don't really like having to pin pypgstac here, but I didn't wanted an overcomplex solution